README.md

```diff
@@ -1,5 +1,82 @@
+- Next there's a series of steps defined by each of these "rules". Each of these steps has their input, output, and separate conda environment, parameters, log file, and shell command to execute that need to be specified.
+
+Each step has these settings (I've described in plain speak what these are generally for).
+```
+input:
+    "This builds together the input file name using the wildcards specified at the start of the file"
+output:
+    "This specifies where the output results files should be stored"
+conda:
+    "This tells us where the conda environment file is for this step so we have the packages we need"
+params:
+    "This is defining the files and folders and other parameters"
+log:
+    "Where the log should be stored"
+shell:
+    "A shell command to be run that has the wildcards that are defined above -- this is what is doing the work"
+```
+### Core steps in the workflow:
+
+In the snakefile you can see where this is called, but if you want to see what is happening in the actual step you need to look at the corresponding Rmd file in the `scripts` folder.
+
+### `scripts/pgRNA_counts_QC.Rmd`
+
+This Rmd runs QC and applies a low count filter
+
+  - It makes a cummulative distribution function
+  - Prints out the Counts per million (CPM) per sample
+  - Does a sample to sample correlation
+  - Flags samples that maybe don't have enough counts for the plasmid
+    - This requries a cutoff to be set -- how low is too low?
+  - Then prints out what was removed.
+- `get_pgRNA_annotations` - This annotates the data
+  - It grabs Entrez Ids and Ensembl annotation
+  - Grabs Copy Number and Transcript per Millions data from a cancer dependency dataset using [depMap](https://bioconductor.org/packages/release/data/experiment/vignettes/depmap/inst/doc/depmap.html#1_introduction)
+  - It labels genes as `negative_control`, `positive_control`, `single_targeting` or `double_targeting`.
+
+### `scripts/calculate_LFC.Rmd`
+
+This Rmd calculates log fold change and makes heatmaps
+
+  - Does filtering based on annotations
+  - Uses custom plotting functions to make heatmaps and violin plots
+  - Calculates  SSMD?
+  - Investigates correlations across replicates
+  - How log fold changed is normalized:
+    - `Take LFC, then subtract median of negative controls. This will result in the median of the nontargeting being set to 0. Then, divide by the median of negative controls (double non-targeting) minus median of positive controls (targeting 1 essential gene). This will effectively set the median of the positive controls (essential genes) to -1.`
+    - `Since the pgPEN library uses non-targeting controls, we adjusted for the fact that single-targeting pgRNAs generate only two double-strand breaks (1 per allele), whereas the double-targeting pgRNAs generate four DSBs. To do this, we set the median (adjusted) LFC for unexpressed genes of each group to zero.`
+  - Does different handling for single level targeting versus double level targeting
```

Comment:
Referencing this point: `Since the pgPEN library uses non-targeting controls, we adjusted for the fact that single-targeting pgRNAs generate only two double-strand breaks (1 per allele), whereas the double-targeting pgRNAs generate four DSBs. To do this, we set the median (adjusted) LFC for unexpressed genes of each group to zero.`

Siobhan or Alice might have more insights into this, but I'm not really understanding how these are distinguished analytically (i.e. how does setting the median this way account for the extra DSBs? is each group a paralog pair, and what genes go unexpressed?)

From a biology perspective, I think they're saying the single-KO will generate a DSB and the cell will express two copies of that allele (and thus two DSBs), and dual-KO will generate two DSBs at different loci of the same allele (four total DSBs). I'm not sure if that's the most complete explanation, though